app/celery/scheduled_tasks.py

```diff
@@ -222,6 +222,24 @@ def check_templated_letter_state():
             )
 
 
+@notify_celery.task(name="in-flight-to-inbox-sms")
+@statsd(namespace="tasks")
+def in_flight_to_inbox_sms():
```

Comment:
To keep the internal implementation abstract from the beat process, can we remove mentions of in-flights and inbox as much as we can? We rename with the intent: we want to time out the notifications that were left out, i.e. `recover_lost_notifications`.